src/events/order/cd-event-add-child-ticket.vue

```diff
@@ -18,10 +18,10 @@
         <vue-dob-picker select-class="form-control cd-child-ticket__dob-picker-wrapper-select" v-model="dob"
                         show-labels="false" month-format="short"
                         :placeholders="[$t('Date'), $t('Month'), $t('Year')]"
-                        :proportions="[2, 2, 3]" data-vv-value-path="value" :data-vv-name="`dob-${id}`" v-validate="'required'"></vue-dob-picker>
+                        :proportions="[2, 2, 3]" data-vv-value-path="value" :data-vv-name="`dob-${id}`" v-validate="'required|overage'"></vue-dob-picker>
       </div>
       <p class="cd-child-ticket__dob-err text-danger" v-show="errors.has(`dob-${id}:required`)">{{ $t('Date of birth is required') }}</p>
-      <p class="cd-child-ticket__dob-u19-err text-danger" v-show="errors.has(`dob-u19-${id}:overage`)">{{ $t('Youths need to be 18 years old or younger.') }}</p> 
+      <p class="cd-child-ticket__dob-u18-err text-danger" v-show="errors.has(`dob-${id}:overage`)">{{ $t('Youth tickets are for attendees 18 years or younger. No ticket is required for guardians or parents.') }}</p>
```

Comment:
All the checks are for them being under 18, not 18 and under. The checks are fine, this should be changed to "attendees under 18 years" or "attendees 17 years or younger".

Reply:
gotcha